crates/proto/src/version/v662/packets/mod.rs

```diff
@@ -7,12 +7,50 @@ macro_rules! export {
 
 export!(login);
 export!(play_status);
-export!(handshake_server_to_client);
-export!(handshake_client_to_server);
+export!(server_to_client_handshake);
+export!(client_to_server_handshake);
```

Comment:
I would prefer if you would rename `server_to_client_handshake` and `client_to_server_handshake` to `handshake_server_to_client` and `handshake_client_to_server`, thanks :)

Reply:
sounds a bit weird, does it have to be?